src/RoutingControllersOptions.ts

```diff
@@ -1,4 +1,5 @@
 import {ClassTransformOptions} from "class-transformer";
+import {TransformValdiationOptions} from "class-transformer-validator";
```

Comment:
if users will install `routing-controllers` without `class-transformer-validator` then their typescript compiler will always give error that it can't find `class-transformer-validator` package. So, using this expression you force users to always install `class-transformer-validator`